src/main/java/ch/njol/skript/events/SimpleEvents.java

```diff
@@ -842,5 +767,15 @@ public class SimpleEvents {
 			)
 			.since("INSERT VERSION");
 
+		Skript.registerEvent("Vehicle Move", SimpleEvent.class, VehicleMoveEvent.class, "vehicle move")
```

Comment:
Is this event called a lot like PlayerMoveEvent/EntityMoveEvent? A disclaimer/warning in the description might not be a bad idea